packages/template-phpstan-compiler/composer.json

```diff
@@ -8,7 +8,7 @@
         "nette/utils": "^3.2",
         "symplify/astral": "^10.0",
         "symplify/package-builder": "^10.2",
-        "phpstan/phpstan": "^1.5.6"
+        "phpstan/phpstan": ">=1.5.6 <=1.7.15"
```

Comment:
There should be bumped to PHPStan 1.8 as well

Reply:
I wanted to make this package working again and then update astral and phpstan to newest versions.

With phpstan 1.8 I got error:
```
PHPStan\Analyser\FileAnalyser::analyseFile(): Argument #4 ($collectorRegistry) must be of type PHPStan\Collectors\Registry, null given, called in LatteCompleteCheckRule.php
```

Reply:
That must be upgraded then. We'll use the latest PHPStan 1.8 along with Symplify and Rector.

Reply:
ok phpstan is bumped to 1.8, but what should I do with missing Symplify\Astral\NodeFinder\SimpleNodeFinder? Are there some replacements?

Reply:
See the PHPStan release info: https://phpstan.org/blog/preprocessing-ast-for-custom-rules

For some cases, custom node visitor is needed.
In other case, the rule can be rewritten not to need the parent/next nodes.